CONTRIBUTING.md

```diff
@@ -50,3 +63,7 @@ For example:
 niv update nixpkgs -r 4e329926df7ee5fa49929a83d31ee7d541f8b45c
 niv update nixpkgs -v 21.11.337905.902d91def1e
 ```
+
+
+[Agda]: https://wiki.portal.chalmers.se/agda/pmwiki.php
```

Comment:
I just put the links at the bottom to make the plain text version of this file easier for us to read/edit.